Source/WebKit/WebKit.xcodeproj/project.pbxproj

```diff
@@ -18853,7 +18855,6 @@
 			isa = PBXTargetDependency;
 			platformFilters = (
 				ios,
-				xros,
```

Comment:
This and the other 3 xros subtractions seem errant.

Reply:
Yeah, not sure what Xcode was doing...